app/main.py

```diff
@@ -14,4 +14,7 @@ def count_occurrences(phrase: str, letter: str) -> int:
     :param letter: letter to find occurrences of it
     :return: count occurrences of letter in phrase
     """
-    # write your code here
+
+    lowercase_phrase = phrase.lower()
+    lowercase_letter = letter.lower()
```

Comment:
There is no need for these variables, use [chaining](https://stackoverflow.com/questions/41817578/basic-method-chaining) to perform all operations in one line in the `return` statement.